src/components/Footer.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react'
```

Comment:
Let's open a issue to change this in the feature, to make the footer fixed to the bottom by default.